spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/ExecutorBeanPostProcessor.java

```diff
@@ -81,7 +83,12 @@ public Object postProcessAfterInitialization(Object bean, String beanName)
 			boolean classFinal = Modifier.isFinal(bean.getClass().getModifiers());
 			boolean cglibProxy = !classFinal;
 			ThreadPoolTaskExecutor executor = (ThreadPoolTaskExecutor) bean;
-			return createThreadPoolTaskExecutorProxy(bean, cglibProxy, executor);
+			AsyncProperties asyncProperties = asyncConfigurationProperties();
```

Comment:
We need a test for that

Reply:
I can add the test. What part do you want to exactly test?

Reply:
```
given: a `spring.sleuth.async.ignoredBeans: fooExecutor `property was set
and: a `fooExecutor` bean exists
when: `fooExecutor` is executed
then: there is no span reported to zipkin
```